tests/niminaction/Chapter2/various2.nim

```diff
@@ -140,7 +140,7 @@ let numbers = @[1, 2, 3, 4, 5, 6]
 let odd = filter(numbers, proc (x: int): bool = x mod 2 != 0)
 doAssert odd == @[1, 3, 5]
 
-import sequtils, future
+import sequtils, sugar
```

Comment:
You shouldn't change that, so the PR can't really be merged :)
Nim in Action should stay 100% as-is as it was in the book so it's tested that it's backwards compatible.

Reply:
CC @dom96 so he can explain in more detail :P

Reply:
Indeed. This module is already deprecated, no point in deleting it except to just break my book.